Client/Application/AppDelegate.swift

```diff
@@ -17,10 +18,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     private let log = Logger.browserLogger
 
-    lazy var profile: Profile = BrowserProfile(
-        localName: "profile",
-        syncDelegate: UIApplication.shared.syncDelegate
-    )
+    lazy var profile: Profile = BrowserProfile(localName: "profile",
```

Comment:
nano-nit - this indentation seems strange - having all args on their own line and with a standard indent seems more consistent with other code and less prone to requiring all lines needing reindent just because the first line changes.